.github/workflows/basic.yml

```
@@ -7,8 +7,19 @@ on: [workflow_call, workflow_dispatch]
 
 jobs:
 
+  workflowcheck:
+    name: Check validity of GitHub workflows
+    runs-on: ubuntu-latest
+    container: openquantumsafe/ci-ubuntu-latest:latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4@692973e3d937129bcbf40652eb9f2f61becf3332 # pin@v4
```

```suggestion
        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # pin@v4
```
I think this will fix the problem.